presto-main/src/main/java/com/facebook/presto/transaction/InMemoryTransactionManager.java

```diff
@@ -573,12 +573,14 @@ public synchronized void checkConnectorWrite(ConnectorId connectorId)
         private synchronized ListenableFuture<?> abortInternal()
         {
             // the callbacks in statement performed on another thread so are safe
-            return nonCancellationPropagating(Futures.allAsList(Stream.concat(
```

Comment:
Unrelated change?

Reply:
Will put a separate diff if required. For some reason, this would not allow Idea to compile the code.